lib/iris/tests/unit/cube/test_Cube.py

```diff
@@ -409,6 +409,155 @@ def test_kwargs(self):
         self.assertMaskedArrayEqual(expected_result, res_cube.data)
 
 
+class TestSlices(tests.IrisTest):
```

Comment:
By the naming rules, I think this should be called `Test_slices`.
Or in fact, as it only really tests one aspect of behaviour, you may as well call it `Test_slices__dim_order` or somesuch.

If you really wanted, you could be more comprehensive -- I that see Test_slices_over is pretty involved(!).
Why would you want to do that, though ?? I'm certainly not asking for it.